### PR TITLE
Add class to wrap result fields into lists for evaluation

### PR DESCRIPTION
**Pull request by melikovk**
This class is useful to create an evaluation pipeline that does not rely on 
MultiScaleFlipAugmentation 

On branch wrap_into_lists
Changes to be committed:
modified:   mmdet/datasets/pipelines/formating.py

### CODE REVIEW

**Comment by yhcao6** [comment]
Thanks for the pr. Could you show a usage example?

**Comment by melikovk** [comment]
Here is an example test pipeline usage
`test_pipeline = [
    dict(type='LoadImageFromFile', color_type='unchanged', to_float32=True),
    dict(type='Normalize', **img_norm_cfg),
    dict(type='Pad', size_divisor=32),
    dict(type='ImageToTensor', keys=['img']),
    dict(type='Collect', keys=['img']),
    dict(type='WrapIntoLists')
]`

File: mmdet/datasets/pipelines/formating.py [hunk 1]
```diff
@@ -190,3 +190,16 @@ def __call__(self, results):
     def __repr__(self):
         return self.__class__.__name__ + '(keys={}, meta_keys={})'.format(
             self.keys, self.meta_keys)
+
+
+@PIPELINES.register_module
+class WrapFieldsToLists(object):
```
Comment (hellock) [inline mmdet/datasets/pipelines/formating.py]: Docstring is missing.

Comment (melikovk) [reply mmdet/datasets/pipelines/formating.py]: I have added a docstring

**Pull request merged**